etc/cli.angular.io/license.html

```diff
@@ -0,0 +1,23 @@
+<pre style="word-wrap: break-word; white-space: pre-wrap;">The MIT License
+
+Copyright (c) 2014-2016 Google, Inc.
```

Comment:
is this the same way it's licensed in the original?

Reply:
Yes, I only copied the file over. I'll add a commit to remove the years.

Reply:
Note that the license file isn't pointed at anywhere. It was just part of the original website so I published it verbatim. Seems like all microsites have one (also with dates): https://universal.angular.io/license.html